src/components/styled/StyledPagination.tsx

```diff
@@ -0,0 +1,37 @@
+import { Pagination, PaginationProps } from "@mui/material";
+import { styled } from "@mui/material/styles";
+import theme from "theme";
+
+const grey100 = theme.colors.grey100;
+
+const StyledPagination = styled(Pagination)<PaginationProps>({
+  color: grey100,
+
+  "& .MuiPaginationItem-root": {
+    color: grey100,
+    borderRadius: "0px",
+    border: `1px solid ${grey100}`,
+    margin: "0px",
+  },
+  "& .MuiPaginationItem-previousNext > svg": {
+    color: grey100,
+  },
+  "& .MuiPaginationItem-previousNext": {
+    borderRadius: "0px 6px 6px 0px",
+  },
+  "& li:first-child > button": {
+    borderRadius: "6px 0px 0px 6px",
+  },
+
+  "& .MuiPaginationItem-ellipsis": {
+    paddingTop: "5px !important",
+    paddingBottom: "5px !important",
```

Comment:
Is there a way that we can increase specificity to the selector so that we dont need to add the important here? 

Also, can these padding values be 4px instead of 5px? As we add more content to the pages, following the 4 point grid system will help with the vertical rhythm of the page and prevent alignment issues in the future

Reply:
I think I accidentally left those !important flags because it works fine without them. Also, good point about the padding, lets's just make sure that design is aware of the 4-point grid system.